cmd/gh/main.go

```diff
@@ -40,6 +40,7 @@ const (
 	exitCancel  exitCode = 2
 	exitAuth    exitCode = 4
 	exitPending exitCode = 8
+	exitUnkown  exitCode = 16
```

Comment:
I'm not particularly concerned about what exit code to use for this situation, so happy to hear others' thoughts about it if you think we could use something better.

Reply:
Or even one of the existing exit codes. Though I will say, making a unique exit code for an "unknown results" situation does make the most sense to me. Willing to be convinced otherwise though!